src/condor_utils/read_multiple_logs.cpp

```diff
@@ -182,7 +176,7 @@ ReadMultipleUserLogs::GetLogStatus()
 int
 ReadMultipleUserLogs::totalLogFileCount() const
 {
-	return allLogFiles.getNumElements();
+	return (int)allLogFiles.size();
```

Comment:
How much work would it be to propagate this return value to be a `size_t` all the way up?

Reply:
I hadn't looked upstream. I like the idea of doing that if it's not too much work.